### PR TITLE
Beef up the readme

### PR DESCRIPTION
**Pull request by BintLopez**
https://trello.com/c/Us0avQyc

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,136 +1,90 @@
...
-
-Install dependencies, including [Botkit](https://github.com/howdyai/botkit):
-
+# Pigeon
+Insert test build badge here. (Will be addressed in this task - https://trello.com/c/yptFCLY3)
+
+## How it Works
+Pigeon is a case management Slack app that allows volunteers to manage a text/chat hotline. Clients text a hotline number, Pigeon opens up a case and a conversation with the client via a chat channel.
+
+Try it out by joining our [sandbox pigeon slack here](want-to-link-to-CSF-hotline-slack).
```
Comment (BintLopez) [inline readme.md]: Motion to use csf-hotline.slack.com as the sandbox slack for potential users to join and test out cases. Thoughts?

File: readme.md [hunk 2]
```diff
@@ -1,136 +1,90 @@
...
+  * `/assign` - assigns a case to a user
+  * `/flag` - flags a case any label you write, "needs attention" is the default
+  * `/unflag` - unflags a case
+  * `/getflags` - returns a list of all flags
+  * `/success` - closes a case
+  * `/logout` - unassigns all cases for that user (ends their volunteer shift)
+
+## How to Develop Locally
+
+The jdp-hotline-test slack is pointed at an ngrok generated URL which routes to port 3000, where our local app instance is running. If you are not part of the jdp-hotline-test slack, please request an invite by filing an issue and tagging @DDF-dev.
```
Comment (BintLopez) [inline readme.md]: I made a DDF-dev team -- figure issue tagging us would be a good way to do access requests?

File: readme.md [hunk 3]
```diff
@@ -1,136 +1,94 @@
...
+  * `/success` - closes a case
+  * `/logout` - unassigns all cases for that user (ends their volunteer shift)
+
+## How to Develop Locally
+
+In order to develop locally without creating a Slack app from scratch, you'll need to request access for a few things:
+* Access to jdp-hotline-test.slack.com
+* Access to jdp-wilson-test app
+* Ngrok account (for using a subdomain locally)
+Please request access to these items by opening an issue and tagging @DDF-dev.
```
Comment (BintLopez) [inline readme.md]: ^^ thoughts on this process?

**Comment by BintLopez** [comment]
@WCRest -- another PR for your reviewing 👀 👁 👁‍🗨 👓 👀

File: readme.md [hunk 4]
```diff
@@ -1,136 +1,94 @@
...
+
+In order to develop locally, you'll need to run a local instance of the app and use ngrok to point a public URL to your local port. The jdp-hotline-test slack is set up to point to the ngrok generated URL. Below are steps to get started:
```
Comment (BintLopez) [inline readme.md]: Should add -- "Note -- this will only allow one person to develop locally at a time"

File: readme.md (hunk 3 above)
Comment (WCRest) [reply readme.md]: Looks good! Lets change the name of jdp-wilson-test app?

**Comment by WCRest** [comment]
lgtm

**Pull request merged**